src/UnitTests/IMappingExpression/ForCtorParam.cs

```diff
@@ -51,5 +55,52 @@ public void Should_resolve_using_custom_func()
 
             dest.Value1.ShouldEqual(8);
         }
+
+        [Fact]
+        public void Should_ignore_nonexistent_parameter()
+        {
+            var config = new MapperConfiguration(cfg =>
+            {
+                cfg.CreateMap<Source, Dest>()
+                    .ForCtorParam("thing", opt => opt.MapFrom(src => src.Value))
+                    .ForCtorParam("think", opt => opt.MapFrom(src => src.Value));
```

Comment:
This should throw. I see the other mapping, but if the parameter doesn't exist, this should be reported somehow.